Sources/AblyAssetTrackingPublisher/PublisherBuilder.swift

```diff
@@ -55,4 +55,9 @@ public protocol PublisherBuilder {
      It's disabled by default
      */
     func rawLocations(enabled: Bool) -> PublisherBuilder
+    
+    /**
+     Set constant resolution for locatoin engine
+     */
+    func constantLocationEngineResolution(resolution: Resolution?) -> PublisherBuilder
```

Comment:
why nullable but rawLocations not?

Reply:
> why nullable but rawLocations not?

it's taken from the android implementation, so the question is rather for @KacperKluka than for me.

>  (Also, does it make sense for areRawLocationsEnabled to be optional - should it not just be a boolean?

https://github.com/ably/ably-asset-tracking-swift/pull/266#discussion_r814229688

> Both areRawLocationsEnabled and constantLocationEngineResolution are optional in DefaultPublisher, but the PublisherBuilder param for the former is optional and for the latter not. Should they be consistent?

You're right, but since this PR concerns the "constant location engine resolution", I'll change it in a separate PR.

Reply:
> why nullable but rawLocations not?

That's because `rawLocations` can be only enabled or disabled so there's no point in adding the third state `null` which would probably mean disabled. With the `constantLocationEngineResolution` the thing is different as we either want to set the constant resolution or disable it, therefore, we've decided that sending a `null` means that it should be disabled :wink:

Reply:
Cool, makes sense. Sounds like the the issue is more likely with `rawLocations` than with this new property, so let's discuss it in https://github.com/ably/ably-asset-tracking-swift/pull/284.